tools/runner/runner.go

```diff
@@ -141,11 +141,15 @@ func (r *Runner) runTest(ctx context.Context, config *grpcv1.LoadTest, reporter
 		status = statusString(config)
 		switch {
 		case loadTest.Status.State.IsTerminated():
-			_, err := r.logSaver.SavePodLogs(ctx, loadTest, outputDir)
+			podToPathMap, err := r.logSaver.SavePodLogs(ctx, loadTest, outputDir)
 			if err != nil {
 				reporter.Error("Could not save pod logs: %s", err)
 			}
 			reporter.AddProperty("name", loadTest.Name)
+			for pod := range podToPathMap {
+				propertyKey := "pod." + strings.TrimPrefix(pod.Name, loadTest.Name+"-") + ".name"
```

Comment:
As a secondary note, you probably want to have a function that generates a "pod to property name element" for the pod (just the "client-0" and so on) and functions that generate property name and property log keys from this element. I wouldn't be doing + + but either fmt.Sprintf or strings.Join in those functions, and I wouldn't put them in runner.go either.